src/locales/la.ts

```diff
@@ -0,0 +1,797 @@
+import { RomcalLocale } from '@romcal/models/romcal-locale';
+
+export default {
+  advent: {
+    season: 'Adventus',
+    feria: '{{day}}, hebdomada {{week}} Adventus',
+    sunday: 'Dominica {{week}} Adventus',
+  },
+  christmastide: {
+    season: 'Nativitatis',
+    day: '{{day}} of Christmastide', // TODO: Where is this used?
+    octave: 'De die {{count}} infra octavam Nativitatis',
+    sunday: 'Dominica {{count}} post Nativitatem',
+  },
+  epiphany: {
+    season: 'Epiphaniæ',
+    before: '{{day}} post octavam Nativitatis',
+    after: '{{day}} post Epiphaniam',
+  },
+  ordinaryTime: {
+    season: 'Per annum',
+    feria: '{{day}}, hebdomada {{week}} per annum',
+    sunday: 'Dominica {{week}} per annum',
+  },
+  lent: {
+    season: 'Quadragesimæ',
+    feria: '{{day}}, hebdomada {{week}} Quadragesimæ',
+    sunday: 'Dominica {{week}} Quadragesimæ',
+    dayAfterAshWed: '{{day}} post Cineres',
+  },
+  holyWeek: {
+    season: 'Hebdomadæ Sanctæ',
+    feria: '{{day}} Hebdomadæ Sanctæ',
+  },
+  eastertide: {
+    season: 'Paschæ',
+    feria: '{{day}}, hebdomada {{week}} temporis paschalis',
+    sunday: 'Dominica {{week}} Paschæ',
+    octave: 'De die {{count}} infra octavam Paschæ',
```

Comment:
Replace `{{count}}` by `{{day}}` and this localization should work properly.

Reply:
Thanks! I have somewhat copied the `{{count}}` from the `christmastide.octave` value.